scripts/scripts.js

```diff
@@ -268,6 +268,18 @@ function autolinkModals(element) {
       openModal(origin.href);
     }
   });
+
+  const config = {
+    path: getMetadata('trigger-modal-path'),
```

Comment:
```
modal-path
modal-size
modal-trigger
modal-trigger-delay
```

Reply:
`modal-trigger-delay` since this value can be in % or in seconds lets call it `modal-trigger-threshold`
@vdua